openhands/runtime/utils/bash.py

```diff
@@ -275,7 +275,7 @@ def _continue_bash(
                 output += '\r\n' + bash_prompt
         return output, exit_code
 
-    def run(self, action: CmdRunAction) -> CmdOutputObservation | FatalErrorObservation:
+    def run(self, action: CmdRunAction) -> CmdOutputObservation | ErrorObservation:
```

Comment:
**Suggestion:** Ensure consistent return types in the `run` method.

**Existing Code:**
```Python
def run(self, action: CmdRunAction) -> CmdOutputObservation | FatalErrorObservation:
```

**Improved Code:**
```Python
def run(self, action: CmdRunAction) -> CmdOutputObservation | ErrorObservation:
```

**Details:**
Ensure that the `run` method consistently returns the same type of observation to avoid type-related issues.